client/src/components/ProductDetailPage.jsx

```diff
@@ -93,21 +96,25 @@ class ProductDetailPage extends React.Component {
     });
   }
 
-  onChangeFileHandler(event) {
-    this.setState({
-      selectedImageFile: event.target.files[0],
-      fileLoaded: 0
-    });
-  }
+  // onChangeFileHandler(event) {
```

Comment:
Should these be removed from the product detail page since they are commented out?

Reply:
Yes, thanks! I moved them to the component and forgot to check that nothing else needed them. Everything seems to work, so I'll remove!